activity_browser/layouts/tabs/activity.py

```diff
@@ -349,3 +343,9 @@ def save_splitter_state(self):
             self.group_splitter.saveState()
         ).hex()
         ab_settings.write_settings()
+
+    def open_properties(self):
```

Comment:
This is a really weird naming convention - it at least needs to be commented on why this is called "open_..."

Reply:
open_property_editor ?

Reply:
Sure, that works.